render_pdf/src/make_latex.py

```diff
@@ -0,0 +1,92 @@
+from typing import Union
+import parser
+import os
+import jinja2
+
+# TODO probably best to use pydantic for a config template for the dict below. 
```

Comment:
Or a dataclass since it's already in the standard lib:

https://docs.python.org/3/library/dataclasses.html

Reply:
Might be better to do this earlier rather than later before the logic is too baked in. I suggest:

```
@dataclass
class Metadata:
  title: str
  author: str
  language: str
  body: list
  analysis: list
```

then, further below:

```
data = Metadata(
  title=title.lower(),
author=author.lower()
analysis=analysis,
...
)
```